integration-libs/opf/base/components/opf-cta/opf-cta-element/opf-cta-element.component.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  inject,
+} from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+
+@Component({
+  selector: 'cx-opf-cta-element',
+  templateUrl: './opf-cta-element.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OpfCtaElementComponent {
+  protected sanitizer = inject(DomSanitizer);
+
+  @Input() ctaScriptHtml: string;
+
+  renderHtml(html: string): SafeHtml {
+    return this.sanitizer.bypassSecurityTrustHtml(html);
```

Comment:
same as Checkout page , no choice to use byPass, if CTA html is an iframe tag or contains some onclick events...

Reply:
great! thx Roman